mmdet/datasets/dataset_wrappers.py

```diff
@@ -358,10 +358,10 @@ def __getitem__(self, idx):
                 ]
                 results['mix_results'] = mix_results
 
-            if self._dynamic_scale is not None:
-                # Used for subsequent pipeline to automatically change
-                # the output image size. E.g MixUp, Resize.
-                results['scale'] = self._dynamic_scale
+            # if self._dynamic_scale is not None:
```

Comment:
Should we remove these comments lines?

Reply:
Yes. Once we have decided on the plan, we can delete it.